src/main/java/bio/terra/stairway/HookWrapper.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.stairway;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HookWrapper {
+  private static final Logger logger = LoggerFactory.getLogger(HookWrapper.class);
+  private Stairway stairway;
+
+  public HookWrapper(Stairway stairway) {
+    this.stairway = stairway;
+  }
+
+  public HookAction startFlight(FlightContext context) {
+    return handleHook(context, HookType.startFlight);
+  }
+
+  public HookAction endFlight(FlightContext context) {
+    return handleHook(context, HookType.endFlight);
+  }
+
+  public HookAction startStep(FlightContext context) {
+    return handleHook(context, HookType.startStep);
+  }
+
+  public HookAction endStep(FlightContext context) {
+    return handleHook(context, HookType.endStep);
+  }
+
+  private HookAction handleHook(FlightContext context, HookType type) {
+    if (stairway.getStairwayHook() == null) {
+      return HookAction.CONTINUE;
```

Comment:
I wasn't sure if we would would this to return fault or continue. I was thinking that we only wanted "fault" to come from the return of the hook, not to signal an error within stairway.

Reply:
We might want to have a separate return, like `IS_NULL`, or if we use `CONTINUE` then set a boolean that can be used to signal that the `null` condition was reached.

Reply:
I don't know what IS_NULL info would be used for. I would leave it as is and plumb in an alternate return if/when that comes up.